### PR TITLE
Feat:added category

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestion
categories

### CODE REVIEW

**Review by stutisk — APPROVED** [review]
Clean code ! Can go forward with merging it

File: index.html [hunk 1]
```diff
@@ -71,6 +71,39 @@
                 src="https://www.decathlon.in/_next/image?url=https%3A%2F%2Fcontents.mediadecathlon.com%2Fs896860%2Fk%24382f8943451fc70c7f2ae3d4ab3db60a%2Fdesktop.jpg%3Ff%3D0x250%3Ff%3D1920x470%26format%3Dauto&w=1920&q=75" />
         </section>
 
+        <section class="category-nav">
+         <section class="category">
```
Comment (sanathkumar26700) [inline index.html:75]: Suggestion : could use article/div instead of section to help browser identify markup semantically.

File: style.css [hunk 2]
```diff
@@ -101,10 +105,47 @@
...
+    width: 5rem;
+    height: 5rem;
+    margin-right: 0.4rem;
+}
 /* -----------container------------------ */
 
 .category-container {
     display: flex;
+    flex-wrap: wrap;
+    row-gap: 20px;
```
Comment (sanathkumar26700) [inline style.css]: usage of rem for consistency

**Review by sanathkumar26700 — APPROVED** [review]

**Comment by prathameshpcodes** [comment]
Go ahead & merge it

**Review by krate27 — COMMENTED** [review]
Use good code formatter

**Pull request merged**